app/helpers/avatar_helper.rb

```diff
@@ -113,12 +113,20 @@ def gravatar_default_options
         :secure => true,
         :default => default_avatar_url
       },
-      :class => 'user-avatar'
+      :class => 'avatar left mrm'
```

Comment:
Use the new Ruby 1.9 hash syntax.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.